hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/file/SqlAvroTest.java

```diff
@@ -187,6 +184,15 @@ public void test_schemaDiscovery() {
 
     @Test
     public void test_tableFunction() {
+        test_tableFunction(AVRO_FILE);
+    }
+
+    @Test
+    public void test_tableFunction_nullableFields() {
+        test_tableFunction(AVRO_NULLABLE_FILE);
+    }
```

Comment:
We are missing test reading `null`s. This reads nullable fields, which contain values.